platform/linux-generic/pktio/dpdk.c

```diff
@@ -1684,8 +1685,12 @@ static int dpdk_open(odp_pktio_t id ODP_UNUSED,
 		return -1;
 	pool_entry = pool_entry_from_hdl(pool);
 
-	if (!dpdk_netdev_is_valid(netdev)) {
-		ODP_ERR("Invalid dpdk netdev: %s\n", netdev);
+	if (!rte_eth_dev_get_port_by_name(netdev, &port_id))
+		pkt_dpdk->port_id = port_id;
+	else if (dpdk_netdev_is_valid(netdev))
+		pkt_dpdk->port_id = atoi(netdev);
+	else {
+		ODP_ERR("Invalid interface name!: %s\n", netdev);
```

Comment:
In linux-generic implementation a user may want to use also other interface types than DPDK, so maybe it  would be better to print:
`ODP_DBG("Invalid DPDK interface name: %s\n", netdev);`

Reply:
Updated the print statement